Specs/DateTime/Chinese/DurationParser.json

```diff
@@ -98,7 +98,7 @@
     },
     "Results": [
       {
-        "Text": "三年",
+        "Text": "三年半",
```

Comment:
Can you add a case like this as a full sentence in DateTimeModel.json too?

Reply:
Can you also add a sentence here that has two examples of such cases in the same sentence, to show that the current solution doesn't misbehave in such situations?

Reply:
Same for an example with two durations in DateTimeModel.json

Reply:
I've add some unit test cases both on DateTimeModel.json and DurationParser.json.